<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-16T11:13:04 IST" name="apiSuite" finished-at="2023-08-16T11:13:12 IST" duration-ms="8628">
    <groups>
    </groups>
    <test started-at="2023-08-16T11:13:04 IST" name="apiTest" finished-at="2023-08-16T11:13:12 IST" duration-ms="8628">
      <class name="com.test.testcases.UserTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.test.testcases.UserTest@3023df74]" started-at="2023-08-16T11:13:04 IST" name="setUp" finished-at="2023-08-16T11:13:04 IST" duration-ms="15" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="create_user(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.test.testcases.UserTest@3023df74]" started-at="2023-08-16T11:13:05 IST" name="create_user" data-provider="getdata" finished-at="2023-08-16T11:13:09 IST" duration-ms="3703" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[4357]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[G]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[H]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[ram@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Ram@123]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[7843564555]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [201] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [201] but found [200]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at com.test.testcases.UserTest.create_user(UserTest.java:55)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- create_user -->
        <test-method signature="create_user(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.test.testcases.UserTest@3023df74]" started-at="2023-08-16T11:13:09 IST" name="create_user" data-provider="getdata" finished-at="2023-08-16T11:13:10 IST" duration-ms="1165" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[9958]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ramesh]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[T]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[R]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[rarm@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Ram@124]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[7843564588]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [201] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [201] but found [200]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at com.test.testcases.UserTest.create_user(UserTest.java:55)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- create_user -->
        <test-method signature="get_user(java.lang.String)[pri:2, instance:com.test.testcases.UserTest@3023df74]" started-at="2023-08-16T11:13:10 IST" name="get_user" data-provider="getUsername" finished-at="2023-08-16T11:13:11 IST" duration-ms="1142" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ram]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- get_user -->
        <test-method signature="get_user(java.lang.String)[pri:2, instance:com.test.testcases.UserTest@3023df74]" started-at="2023-08-16T11:13:11 IST" name="get_user" data-provider="getUsername" finished-at="2023-08-16T11:13:12 IST" duration-ms="1144" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ramesh]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- get_user -->
      </class> <!-- com.test.testcases.UserTest -->
    </test> <!-- apiTest -->
  </suite> <!-- apiSuite -->
</testng-results>
